docs/user-guide/config-cert-instance-hints.md

```diff
@@ -0,0 +1,54 @@
+Hints about creating the certificate and instance when configuring Zowe.
```

Comment:
Why is this information not going in the existing doc pages for setting up certificates and instances?

Reply:
It's here as a placeholder - a basis for discussion.  I hope it will be edited into the relevant section of the doc in due course.